packages/core/src/types/VideoViewportProperties.ts

```diff
@@ -0,0 +1,17 @@
+import { ViewportProperties } from './ViewportProperties';
+import Point2 from './Point2';
+
+/**
+ * Stack Viewport Properties
+ */
+type VideoViewportProperties = ViewportProperties & {
+  loop?: boolean;
+  muted?: boolean;
+  pan?: Point2;
```

Comment:
having pan here is odd to me what does this mean?

Reply:
for pan and parallelScale we should treat it similar to the viewport which we have setPan and setZoom programmetically